data/tooltips.txt

```diff
@@ -155,6 +155,9 @@ tip "jump speed:"
 tip "jump fuel:"
 	`How much fuel is consumed when you jump between systems using this drive.`
 
+tip "maintenance cost:"
```

Comment:
My impression is that "maintenance costs" is usually written with an s on the end, but I'm not a native speaker, so please verify if there should be an additional s.

Reply:
Well this is a cost for a single outfit, so just "cost" sounds alright to me, but if others think it should be changed that I'll do it.